requirements-full.txt

```diff
@@ -218,13 +218,12 @@ tenacity==8.0.1; python_version >= "3.6"
 tensorboard-data-server==0.6.1; python_version >= "3.6"
 tensorboard-plugin-wit==1.8.0; python_version >= "3.6"
 tensorboard==2.7.0; python_version >= "3.6"
-tensorflow-estimator==2.6.0
+tensorflow-estimator==2.7.0
 tensorflow==2.6.0
 termcolor==1.1.0
 terminado==0.12.1; python_version >= "3.6"
 testpath==0.5.0; python_version >= "3.7"
 textwrap3==0.9.2; python_version >= "3.6"
-thepassiveinvestor==1.0.9
```

Comment:
@aia do you know why when you update requirements, **thepassiveinvestor** keeps on being dropped? This is @JerBouma package and is currently being used by the terminal